CHANGES.rst

```diff
@@ -5,6 +5,7 @@
 New Features
 ^^^^^^^^^^^^
 
+- Added ext parameter and property to ImageFileCollection to specify the FITS extension.
```

Comment:
Even though it's not enforced I try to keep lines (if possible also in `rst` files) below 80 characters. Also you lost the pull request number 😄